integrated_channels/degreed/migrations/0027_alter_degreedlearnerdatatransmissionaudit_index_together.py

```diff
@@ -17,7 +17,26 @@ class Migration(migrations.Migration):
                 index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
             ),
         ]
-    else:
+    elif 'postgresql' in db_engine:
+        operations = [
+            migrations.SeparateDatabaseAndState(
+                state_operations=[
+                    migrations.AlterIndexTogether(
+                        name='degreedlearnerdatatransmissionaudit',
+                        index_together={('enterprise_customer_uuid', 'plugin_configuration_id')},
+                    ),
+                ],
+                database_operations=[
+                    migrations.RunSQL(sql="""
+                        CREATE INDEX degreed_dldta_85936b55_idx
```

Comment:
Question: Why not `CREATE INDEX CONCURRENTLY` here?

Reply:
I believe  it will be fresh start of database if anyone will be choosing to go with Postgresql and there will be no pre-populated data. However If I use `CONCURRENTLY` there is a potential if could have error in background which is very unlikely but there is a chance. So I believe it will only add complexity with no benefits